collector/lib/ConnScraper.cpp

```diff
@@ -190,6 +190,60 @@ int ReadHexBytes(const char* p, const char* endp, void* buf, int chunk_size, int
   return i;
 }
 
+struct ProcessInfo {
+  std::string comm;
+  std::string exe;
+  std::string exe_path;
+  std::string args;
+};
+
+static void ReadProcessInfo(const char* process_id, int dirfd, struct ProcessInfo& info) {
```

Comment:
I thought we might put functions that read the process information into a separate class from ConnScraper.

Reply:
Correct, this is the current plan.